src/components/common/Actions.vue

```diff
@@ -264,7 +264,7 @@ export default class Actions extends Vue {
         // clear flag
         this.setHaveChanges(false)
       } catch (error) {
-        console.log('Error on onClickFilePay(): ', error)
+        console.log('Error updating filing in onClickFilePay(): ', error)
```

Comment:
I updated these to distinguish which one is displayed (found while trouble-shooting a unit test).